projects/plugins/jetpack/modules/widgets/search/js/search-widget-admin.js

```diff
@@ -300,10 +300,8 @@
 
 	// When widgets are updated, remove and re-add listeners
 	$( document ).on( 'widget-updated widget-added', function( e, widget ) {
-		widget = $( widget );
-
-		var id = widget.attr( 'id' ),
-			isJetpackSearch = ( id && ( -1 !== id.indexOf( 'jetpack-search-filters' ) ) );
+		var idBase = $( widget ).find('.id_base').val(),
```

Comment:
Does `.id_base` exist pre-WordPress 5.8 too?

Reply:
Yes that's right. The field was added in version 2.x. Tested working in WP 5.7 as well. So it's a compatible way to use.

Reply:
And WP 5.8 Widget Editor uses block but keeps the compatibility by adding these fields from the frontend
https://github.com/WordPress/gutenberg/blob/v9.8.3/packages/edit-widgets/src/blocks/legacy-widget/edit/dom-manager.js#L100-L131